public/_posts/_articles/yearn-finance/yvboost/es.md

```diff
@@ -0,0 +1,99 @@
+---
+title:  "Más impulsos, compuestos y mojitos para ti con yvBOOST"
+image:
+  src: ./cover.jpg
+  width: 1050
+  height: 543
+date: '2021-09-14'
+author: Qosmonot
+translator: RestlessMik3
+---
+
+# Más impulsos, compuestos y mojitos para ti con yvBOOST
+
+### La relación de colaboración de Yearn con [Curve.fi](http://curve.fi/) es fundamental para nuestra infraestructura y fundamental para impulsar tus impulsos.
+
+🔵🤝🌈
+
+_publicación escrita por_ [_qosmonot_](http://twitter.com/qosmonot), un colaborador de Yearn
+
+Muchas vaults de Yearn tienen estrategias basadas en los grupos de Curve y, por lo tanto, usan el token CRV para optimizarlas votando para dirigir las asignaciones de recompensas hacia esos grupos. Yearn ha estado aumentando constantemente su titularidad de tokens CRV para proporcionar recompensas cada vez mayores para esas vaults.
+
+![](image1.jpg?w=1050&h=651)
+
+Muchas vaults de Yearn tienen estrategias basadas en los grupos de Curve y, por lo tanto, usan el token CRV para optimizarlas votando para dirigir las asignaciones de recompensas hacia esos grupos. Yearn ha estado aumentando constantemente su titularidad de tokens CRV para proporcionar recompensas cada vez mayores para esas vaults.
+
+## ¿Qué es CRV?
+
+CRV es el token de utilidad y gobernanza de CurveDAO, la organización descentralizada que ejecuta el protocolo [Curve.fi](http://curve.fi/). El objetivo principal del token CRV es incentivar a los proveedores de liquidez en la plataforma [Curve.fi](http://curve.fi/), así como involucrar a tantos usuarios como sea posible en la gobernanza del protocolo. El token CRV tiene tres usos principales: votar, staking e impulsar. Esas tres cosas requieren que los usuarios "bloqueen por voto" su CRV a cambio de veCRV. veCRV significa CRV con depósito de votos, y simplemente representa CRV bloqueado por un período de tiempo (entre 1 semana y 4 años).
```

Comment:
AS-IS: CRV es el token de utilidad y gobernanza de CurveDAO, la organización descentralizada que ejecuta el protocolo [Curve.fi](http://curve.fi/). El objetivo principal del token CRV es incentivar a los proveedores de liquidez en la plataforma [Curve.fi](http://curve.fi/), así como involucrar a tantos usuarios como sea posible en la gobernanza del protocolo. El token CRV tiene tres usos principales: votar, staking e impulsar. Esas tres cosas requieren que los usuarios "bloqueen por voto" su CRV a cambio de veCRV. veCRV significa CRV con depósito de votos, y simplemente representa CRV bloqueado por un período de tiempo (entre 1 semana y 4 años).

TO-BE: CRV es el token de utilidad y gobernanza de CurveDAO, la organización descentralizada que ejecuta el protocolo [Curve.fi](http://curve.fi/). El objetivo principal del token CRV es incentivar a los proveedores de liquidez en la plataforma [Curve.fi](http://curve.fi/), así como involucrar a tantos usuarios como sea posible en la gobernanza del protocolo. El token CRV tiene tres usos principales: votar, staking e impulsar. Esas tres cosas requieren que los usuarios "bloqueen por voto" su CRV a cambio de veCRV. veCRV significa CRV con derecho a votos, y simplemente representa CRV bloqueado por un período de tiempo (entre 1 semana y 4 años).

Replaced "depósito de votos" by "derechos de voto".

Reply:
Done